src/renderer/screens/exchange/Swap2/Form/FromRow.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+import React from "react";
+import { useSelector } from "react-redux";
+import { Trans, withTranslation } from "react-i18next";
+import type { TFunction } from "react-i18next";
+import Box from "~/renderer/components/Box/Box";
+import Input from "~/renderer/components/Input";
+import { SelectAccount } from "~/renderer/components/SelectAccount";
+import Switch from "~/renderer/components/Switch";
+import Text from "~/renderer/components/Text";
+import { shallowAccountsSelector } from "~/renderer/reducers/accounts";
+import { amountInputContainerProps, selectRowStylesMap } from "./utils";
+import { FormLabel } from "./FormLabel";
+import type { Account } from "@ledgerhq/live-common/lib/types";
+
+type Props = {
+  fromAccount: ?Account,
+  setFromAccount: (?Account) => void,
+  fromAmount: ?number,
```

Comment:
We can polish the types later, but amounts throughout the app are `BigNumber` since we deal with huge precissions

Reply:
This applies to all the pr